### PR TITLE
[TIMOB-24911] Android: Filtering R. files when building Android modules does not work on Windows.

### PR DESCRIPTION
**Pull request by ypbnv**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24911

Change regular expression to handle platform specific paths.

**Description:**
Windows OS only. Regular expression does not match files' path with backslashes, so they are included in the .jar and duplicated.

### CODE REVIEW

File: android/cli/commands/_buildModule.js [hunk 1]
```diff
@@ -1607,7 +1607,7 @@ AndroidModuleBuilder.prototype.packageZip = function (next) {
 			moduleJarArchive.on('error', cb);
 			moduleJarArchive.pipe(moduleJarStream);
 
-			var excludeRegex = /.*\/R\.class$|.*\/R\$(.*)\.class$/i;
+			var excludeRegex = new RegExp('.*\\' + path.sep + 'R\\\.class$|.*\\' + path.sep + 'R\\\$(.*)\\\.class$', 'i');
```
Comment (janvennemann) [inline android/cli/commands/_buildModule.js]: The three backslashes as escape characters are unnecessary here, for example `R\\.class$` already results in in the correct regex `R\.class$`

**Review by janvennemann — CHANGES_REQUESTED** [review]

**Comment by ypbnv** [comment]
@janvennemann Updated.

**Review by cb1kenobi — APPROVED** [review]
CR LGTM.

**Review by janvennemann — APPROVED** [review]

**Comment by garymathews** [comment]
Merged for parity with `6_1_X` and `6_2_X`

**Pull request merged**